Mathlib/Algebra/TrivSqZeroExt.lean

```diff
@@ -738,6 +738,32 @@ def inlHom [Semiring R] [AddCommMonoid M] [Module R M] [Module Rᵐᵒᵖ M] : R
 
 end Mul
 
+section Inv
+
+variable {R : Type u} {M : Type v}
+
+instance inv [Neg M] [Inv R] [SMul Rᵐᵒᵖ M] [SMul R M] : Inv (tsze R M) :=
+  ⟨fun b => (b.1⁻¹,  - (b.1⁻¹ •> (b.2 <• b.1⁻¹)))⟩
+
+theorem fst_inv [Neg M] [Inv R] [SMul Rᵐᵒᵖ M] [SMul R M]
+    (x : tsze R M) : fst x⁻¹ = (fst x)⁻¹ :=
+      rfl
+
+theorem snd_inv [Neg M] [Inv R] [SMul Rᵐᵒᵖ M] [SMul R M] (x : tsze R M) :
+    snd x⁻¹ = - ((fst x)⁻¹ •> (snd x <• (fst x)⁻¹)) :=
+      rfl
+
+theorem TrivEqZeroExt.mul_inv_cancel [DivisionRing R] [AddCommGroup M]
+    [Module Rᵐᵒᵖ M] [Module R M] {x : tsze R M}
+    (hx : fst x ≠ 0) : x * x⁻¹ = 1 := by
+      ext
+      · rw [fst_mul, fst_inv, fst_one, DivisionRing.mul_inv_cancel (fst x) hx]
+      rw [snd_mul, snd_inv, smul_neg, ← smul_assoc]
+      rw [smul_eq_mul, DivisionRing.mul_inv_cancel (fst x) hx]
+      rw [one_smul, fst_inv, add_left_neg, snd_one]
```

Comment:
Per [the style guide](https://leanprover-community.github.io/contribute/style.html), this should be:

```suggestion
    (hx : fst x ≠ 0) : x * x⁻¹ = 1 := by
  ext
  · rw [fst_mul, fst_inv, fst_one, DivisionRing.mul_inv_cancel (fst x) hx]
  · rw [snd_mul, snd_inv, smul_neg, ← smul_assoc]
    rw [smul_eq_mul, DivisionRing.mul_inv_cancel (fst x) hx]
    rw [one_smul, fst_inv, add_left_neg, snd_one]
```
that is: the line after `by` is *less* indented than the statement.